packages/api/src/store/push-target-table.ts

```diff
@@ -0,0 +1,62 @@
+import { PushTarget } from "../schema/types";
+import Table from "./table";
+import { GetOptions, WithId } from "./types";
+import * as uuid from "uuid";
+import { InternalServerError, UnprocessableEntityError } from "./errors";
+
+type DbPushTarget = WithId<PushTarget>;
```

Comment:
Perhaps `DBPushTarget` for consistency with `DBLegacyObject` and whatnot

Reply:
Cool. Update `WithId` to `WithID` as well, since I'm guessing we don't want our data types to develop [consciousness](https://en.wikipedia.org/wiki/Id,_ego_and_super-ego#Id) quite yet.